clang/tools/clang-linker-wrapper/ClangLinkerWrapper.cpp

```diff
@@ -411,6 +411,8 @@ Error runLinker(ArrayRef<StringRef> Files, const ArgList &Args) {
   // Render the linker arguments and add the newly created image. We add it
   // after the output file to ensure it is linked with the correct libraries.
   StringRef LinkerPath = Args.getLastArgValue(OPT_linker_path_EQ);
+  if (LinkerPath.empty())
+    return createStringError("Host linker is not available");
```

Comment:
Should this go upstream?

Reply:
Yes. Happening in parallel.

https://github.com/llvm/llvm-project/pull/113613#issuecomment-2436215580

tests are different, but expect to converge after pulldown.

Thanks